lua/luaconf.h

```diff
@@ -842,18 +842,18 @@ extern void longjmp(luai_jmpbuf);
 #include <linux/ktime.h>
 #if LINUX_VERSION_CODE < KERNEL_VERSION(5,6,0)
 #include <linux/time.h>
-typedef struct timespec time_t;
+typedef struct timespec lunatik_time;
```

Comment:
```suggestion
typedef struct timespec timespec_t;
```

Reply:
Actually, I agree with @tammela.
I was waiting to see if anybody had an objection to his suggestion.